File: iOS/Bet/Sources/Bet/Pending/PoolGamblerBetModelTransformer.swift
extension PoolGamblerBetModel {
    func toPartialPoolGamblerBet() -> PartialPoolGamblerBetModel {
        PartialPoolGamblerBetModel(
            homeTeamBet: self.homeTeamBetRawValue(),
            awayTeamBet: self.awayTeamBetRawValue()
        )
    }
}

File: Android/bet/src/main/java/com/felipearpa/tyche/bet/pending/PoolGamblerBetModelTransformer.kt
package com.felipearpa.tyche.bet.pending

import com.felipearpa.tyche.bet.PoolGamblerBetModel
import com.felipearpa.tyche.bet.awayTeamBetRawValue
import com.felipearpa.tyche.bet.homeTeamBetRawValue

fun PoolGamblerBetModel.toPartialPoolGamblerBetModel() =
    PartialPoolGamblerBetModel(
        homeTeamBet = this.homeTeamBetRawValue(),
        awayTeamBet = this.awayTeamBetRawValue(),
    )
